api/src/paths/funding-sources/index.ts

```diff
@@ -133,6 +137,36 @@ export function getFundingSources(): RequestHandler {
   };
 }
 
+/**
+ * Checks if the system user is an admin (has an admin level system role).
+ *
+ * @param {SystemUser} systemUserObject
+ * @return {*}  {boolean} `true` if the user is an admin, `false` otherwise.
+ */
+function isAdmin(systemUserObject: SystemUser): boolean {
```

Comment:
Would this function be more useful in a generic place? So other endpoints could utilize it if needed.

Reply:
Good call! Made this a static method in `UserService`.